packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -9,9 +9,28 @@ versionLinks:
   v2: components/decorators/
 ---
 
-Decorators are special functions that can modify the behavior of classes, class methods, and class fields. Lit uses decorators to provide declarative APIs for things like registering elements, reactive properties, and queries.
+Decorators are functions that can be used to declaratively annotate and modify the behavior of classes.
 
-Decorators are a [stage 3 proposal](https://github.com/tc39/proposal-decorators) for addition to the ECMAScript standard. Currently no browsers implement decorators, but compilers like [Babel](https://babeljs.io/) and [TypeScript](https://www.typescriptlang.org/) provide support for an earlier version of the decorators proposal. Lit decorators work with Babel and TypeScript, and will be updated to work with the final specification when it's implemented in browsers.
+Lit provides a set of decorators that enable declarative APIs for things like registering elements, defining reactive properties and query properties, or adding event options to event handler methods.
+
+For example, the `@customElement` and `@property()` decorators let you register a custom element and define a reactive property in a compact, declarative way:
+
+```ts
+@customElement('my-element')
+export class MyElement extends LitElement {
+
+  @property() greeting = 'Welcome';
+
+}
+```
+
+Decorators are a [stage 3 proposal](https://github.com/tc39/proposal-decorators) for addition to the ECMAScript standard. Compilers like [Babel](https://babeljs.io/) and [TypeScript](https://www.typescriptlang.org/) support decorators, though no browsers have implemented them yet. Lit decorators work with Babel and TypeScript, and will work in browsers when they implement them natively.
+
+Previous versions of the decorators proposal are also supported by some compilers, most notably TypeScript's "experimental decorators" which Lit has supported since it's inception.
```

Comment:
```suggestion
Previous versions of the decorators proposal are also supported by some compilers, most notably TypeScript's "experimental decorators" which Lit has supported since its inception.
```

Reply:
Done!